src/routes/solid-start/reference/server/use-server.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: '"use server"'
+---
+
+Perform actions on the server environment only (i.e. console logging, etc.).
+
+```tsx twoslash
+const logHello = async (message: string) => {
+	"use server";
+	console.log(message);
+};
+```
+
+## Basic usage
+
+In the two examples below, regardless of whether we are rendering on the server or in the browser, the `logHello` function generates a log on the server console only. To do this we use compilation to transform the `"use server"` function into an RPC call to the server.
+
+If `"use server"` is inserted as the first line in a file, the entire file will become server-only.
+
+```tsx twoslash {1}
+"use server";
+
+const logHello = async (message: string) => {
+	console.log(message);
+};
+```
+
+If `"use server"` is instead inserted as the first line of a function, only that function will become server-only.
+
+```tsx twoslash {2}
+const logHello = async (message: string) => {
+	"use server";
+	console.log(message);
+};
+
+logHello("Hello");
+```
+
```

Comment:
```suggestion

In both of these examples, the `logHello` function, it would only show in the server console regardless of whether rendering was on the server or in the browser

```